barretenberg/cpp/src/barretenberg/flavor/goblin_ultra_recursive.hpp

```diff
@@ -453,134 +453,7 @@ template <typename BuilderType> class GoblinUltraRecursive_ {
             this->databus_id = verification_key->databus_id;
         }
     };
-
```

Comment:
I'm not sure what this is doing. I don't think we never need to serialize a stdlib transcript, and it was a clashing name, so I removed it.

Reply:
Either way it would be better to reuse the logic from GoblinUltra here so this is fine